db/migrate/20191029181037_add_deploy_builds.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeployBuilds < ActiveRecord::Migration[5.2]
+  def change
+    create_table :deploy_builds do |t|
+      t.references :build, index: true, null: false
+      t.references :deploy, index: true, null: false
+      t.timestamps
```

Comment:
not meant to be updated, so we could drop the updated_at 🤷‍♂ 
```suggestion
      t.timestamp :created_at, null: false
```

Reply:
going to leave it, easier to go with defaults than something custom